src/MVVM/Factories/ViewModelFactory.cs

```diff
@@ -54,19 +54,18 @@ internal static INavigationService TryResolveNavigationServiceInstance()
         }
 
         /// <summary>
-        /// Creates a new ViewModel instance and returns it
-        /// InitializeAsync is invoked in the background
+        /// Creates a new ViewModel instance and optionally invokes the <see cref="IViewModel.InitializeAsync"/> method.
         /// </summary>
-        /// <typeparam name="TViewModelInterface">Type of the ViewModel to create</typeparam>
-        /// <param name="initialize">Should InitializeAsync() get called after getting the instance? <see cref="IViewModel"/></param>
-        /// <returns>ViewModel instance of the given type</returns>
-        public static TViewModelInterface Resolve<TViewModelInterface>(
+        /// <typeparam name="TViewModel">Type of the ViewModel to create</typeparam>
+        /// <param name="initialize">Specifies if <see cref="IViewModel.InitializeAsync"/> should get called after instance creation.</param>
+        /// <returns>A ViewModel instance of the given type</returns>
+        public static TViewModel Resolve<TViewModel>(
```

Comment:
Codacy found an issue: [All 'Resolve' method overloads should be adjacent.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5767060)